pkg/settings/settings.go

```diff
@@ -37,6 +38,7 @@ var (
 	SupportBundleImage                     = NewSetting(SupportBundleImageName, "{}")
 	SupportBundleNamespaces                = NewSetting("support-bundle-namespaces", "")
 	SupportBundleTimeout                   = NewSetting(SupportBundleTimeoutSettingName, "10") // Unit is minute. 0 means disable timeout.
+	SupportBundleExpirationSeconds         = NewSetting(SupportBundleExpirationSecondsSettingName, "1800")
```

Comment:
```
	SupportBundleTimeout                   = NewSetting(SupportBundleTimeoutSettingName, "10") // Unit is minute. 0 means disable timeout.
	SupportBundleExpirationSeconds         = NewSetting(SupportBundleExpirationSecondsSettingName, "1800")
```

Another question: what's the consideration to use different time units here ?

Reply:
Different usage.

- SupportBundleTimeout is used to check if support bundle manager execute too long, such as collecting logs.
- SupportBundleExpirationSeconds is used to remove useless support bundle CR after finished collecting cluster and agent logs.

Reply:
Seems no clear reason that the first setting is in `minutes` but the second is in `seconds`, it may confuse the users.

Could `SupportBundleExpiration` also use minutes ?

Reply:
Surely, then I will remove `Seconds` word.